### ISSUE TITLE
Member edit doesn't submit on enter

### ISSUE THREAD

**Issue by alexdor** [ossn/ossn-frontend#233]
Editing the member's page and hitting enter doesn't submit the changes

**Comment by iropal** [comment]
@alexdor it does submit if the submit button is focused.

**Comment by alexdor** [comment]
I think it should submit even if the submit button isn't focused, at least for me that is the expected ux as a user. Do you think that it's bad ux?\

**Comment by iropal** [comment]
Yes because there are multiple functionalities on this page related to enter and we have the close option for the form as well. I think it would be confusing, especially of we take accessibility into account.

**Comment by alexdor** [comment]
@bacharakis thoughts?
